Assets/Scripts/Game/UserInterfaceWindows/HotkeySequence.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+
+namespace DaggerfallWorkshop.Game.UserInterfaceWindows
```

Comment:
Not a window, should be somewhere else IMO

Reply:
So, I'm thinking of moving this one one level up in DaggerfallWorkshop.Game, as it could be useful in other contexts that the UI, to identify key combos